src/components/filterFooter/FilterFooter.tsx

```diff
@@ -0,0 +1,75 @@
+import classNames from 'classnames';
+import { FilterTypes } from '../../enum/FilterTypes';
+import { Todo } from '../../types/Todo';
+import { FC, useState } from 'react';
+
+interface Props {
+  todos: Todo[];
+  handleFilterChange: (filter: FilterTypes) => void;
+}
+
+export const FilterFooter: FC<Props> = ({ todos, handleFilterChange }) => {
+  const [filter, setFilter] = useState(FilterTypes.All);
+
+  const NotCompletedTodos = todos.filter(todo => !todo.completed).length;
+
+  const setFilterType = (filterType: FilterTypes) => {
+    setFilter(filterType);
+    handleFilterChange(filterType);
+  };
+
+  const isSelectedFilter = (filterType: FilterTypes) =>
+    filter === FilterTypes[filterType];
+
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {NotCompletedTodos} items left
+      </span>
+
+      <nav className="filter" data-cy="Filter">
+        <a
+          href="#/"
+          className={classNames('filter__link', {
+            selected: isSelectedFilter(FilterTypes.All),
+          })}
+          data-cy="FilterLinkAll"
+          onClick={() => setFilterType(FilterTypes.All)}
+        >
+          {FilterTypes.All}
+        </a>
+
+        <a
+          href="#/active"
+          className={classNames('filter__link', {
+            selected: isSelectedFilter(FilterTypes.Active),
+          })}
+          data-cy="FilterLinkActive"
+          onClick={() => setFilterType(FilterTypes.Active)}
+        >
+          {FilterTypes.Active}
+        </a>
+
+        <a
+          href="#/completed"
+          className={classNames('filter__link', {
+            selected: isSelectedFilter(FilterTypes.Completed),
+          })}
+          data-cy="FilterLinkCompleted"
+          onClick={() => setFilterType(FilterTypes.Completed)}
```

Comment:
You can render a list of links by converting your enum into an array with `Object.keys/values/entries` and then map through it